modules/@apostrophecms/schema/ui/apos/components/AposInputRange.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="effectiveError" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <div class="apos-range-input">
+          <input
+            type="range"
+            :min="field.min"
+            :max="field.max"
+            :step="field.step"
+            class="apos-range"
```

Comment:
`apos-range-input__field`? Or `__input`?

Reply:
cleaned all this up to start with `apos-range`